### PR TITLE
Dfr 215 judge name date of order auto populate

### PR DESCRIPTION
**Pull request by GEvans-HMCTS**
Resolves: https://tools.hmcts.net/jira/browse/DFR-215

### CODE REVIEW

File: src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/ApproveConsentOrderContestedAboutToStartHandlerTest.java [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.mockito.Mockito.when;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.CCDConfigConstant.CONSENTED_ORDER_DIRECTION_DATE;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.CCDConfigConstant.CONSENTED_ORDER_DIRECTION_JUDGE_NAME;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.CCDConfigConstant.CONTESTED_ORDER_APPROVED_DATE;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.CCDConfigConstant.CONTESTED_ORDER_APPROVED_JUDGE_NAME;
+
+public class ApproveConsentOrderContestedAboutToStartHandlerTest {
```
Comment (EllisD-B) [inline src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/ApproveConsentOrderContestedAboutToStartHandlerTest.java:29]: Add `@RunWith(MockitoJUnitRunner.class)` to the top of this class (see ManageBarristerServiceTest for reference)

File: src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/ApproveConsentOrderContestedAboutToStartHandlerTest.java [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+    public static final String AUTH_TOKEN = "tokien:)";
+    private ApprovedConsentOrderContestedAboutToStartHandler handler;
```
Comment (EllisD-B) [inline src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/ApproveConsentOrderContestedAboutToStartHandlerTest.java:32]: Add @InjectMocks above your handler

File: src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/ApproveConsentOrderContestedAboutToStartHandlerTest.java [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+    @Mock
+    private IdamService service;
+
+    @Before
+    public void setup() {
+        handler =  new ApprovedConsentOrderContestedAboutToStartHandler(new OnStartDefaultValueService());
```
Comment (EllisD-B) [inline src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/ApproveConsentOrderContestedAboutToStartHandlerTest.java:39]: Remove this setup assignment - you'll also want to mock the OnStartDefaultValueService

File: src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/ApproveConsentOrderAboutToStartHandlerTest.java [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.mockito.Mockito.when;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.CCDConfigConstant.*;
+
+public class ApproveConsentOrderAboutToStartHandlerTest {
+
+    public static final String AUTH_TOKEN = "tokien:)";
```
Comment (EllisD-B) [inline src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/ApproveConsentOrderAboutToStartHandlerTest.java:28]: Same as class below

File: src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/OnStartDefaultValueService.java [hunk 5]
```diff
@@ -2,19 +2,45 @@
...
+import static uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.CCDConfigConstant.CONTESTED_ORDER_APPROVED_DATE;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.CCDConfigConstant.CONTESTED_ORDER_APPROVED_JUDGE_NAME;
 import static uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.CCDConfigConstant.ISSUE_DATE;
 
 @Service
 @RequiredArgsConstructor
 @Slf4j
 public class OnStartDefaultValueService {
 
+    @Autowired
```
Comment (EllisD-B) [inline src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/OnStartDefaultValueService.java:22]: Don't need to autowire the idam service, just make it final, the `@RequiredArgsConstructor` will create a constructor which injects this for you if you mark the Idam service as final

File: src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/ApproveConsentOrderContestedAboutToStartHandlerTest.java (hunk 3 above)
Comment (GEvans-HMCTS) [reply src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/ApproveConsentOrderContestedAboutToStartHandlerTest.java:39]: Was unsure about this as the reference i'm using is the issue application default work that @rishikrsharma had done and he had used this. Happy to mock it however.

**Pull request closed without merging**